app/services/push_notification/http_push.rb

```diff
@@ -38,20 +38,24 @@ def url_options
     def deliver_one(service_provider)
       deliver_local(service_provider) if IdentityConfig.store.risc_notifications_local_enabled
 
-      if AppConfig.env.risc_notifications_sqs_enabled == 'true'
-        deliver_sqs(service_provider)
+      if IdentityConfig.store.risc_notifications_eventbridge_enabled
+        deliver_eventbridge(service_provider)
       else
         deliver_direct(service_provider)
       end
     end
 
-    def deliver_sqs(service_provider)
-      sqs_client.send_message(
-        queue_url: sqs_queue_url,
-        message_body: {
-          push_notification_url: service_provider.push_notification_url,
-          jwt: jwt(service_provider),
-        }.to_json,
+    def deliver_eventbridge(service_provider)
+      eventbridge_client.put_events(
+        entries: [
+          {
+            time: now,
+            source: service_provider.issuer,
+            detail_type: 'notification',
+            detail: jwt(service_provider),
+            event_bus_name: "#{Identity::Hostdata.env}-risc-notifications",
+          },
```

Comment:
```suggestion
            event_bus_name: "#{Identity::Hostdata.env}-risc-notifications",
            trace_header: trace_id,
          },
```

I had a section in like this, but the AWS EventBridge stub rejected all the string values that I put in there, so I am keeping it out for now